pkgs/by-name/li/libsixel/package.nix

```
@@ -7,19 +7,33 @@
   gdk-pixbuf,
   gd,
   pkg-config,
+
+  # Enable linking against image loading libraries as part of the
+  # implementation of the sixel_helper_{load,write}_image_file() functions.
+  # These helper functions are not needed for the main functionality of the
+  # library to encode image buffers to sixels.
+  #
+  # libsixel already uses vendored stb image loading to provide basic
+  # implementations, but also allows for the "gd" library to be linked for
+  # a wider set of image formats.
+  # This pulls in a large amount of deps bloating the resulting library.
+  #
+  # Default off, but configurable in case you really need it.
+  enable_image_loading_gd ? false,
```

```suggestion
  withGd ? false,
```
Would follow the usual naming scheme.

---

Done.